### PR TITLE
Update config.yaml to conform with the revised SageMaker official document

### PR DESCRIPTION
**Pull request by bnsblue**
# TL;DR
Update config.yaml to conform with the revised SageMaker official document

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
The SageMaker official document has been revised to recommend against the use of image tag `:latest` and `:1` for XGBoost, and has deprecated XGBoost 0.72 and the changed the image name from `xgboost` to `sagemaker-xgboost`. This PR updates the config file to conform with the official changes.

image

## Follow-up issue
_NA_

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -65,10 +65,10 @@ plugins:
         regionalConfigs:
           - region: "us-east-1"
             versionConfigs:
-              - version: 0.72
-                image: "811284229777.dkr.ecr.us-east-1.amazonaws.com/xgboost:latest"
-              - version: 0.90
-                image: "683313688378.dkr.ecr.us-east-1.amazonaws.com/xgboost:latest"
+              - version: "0.90"
```
Comment (EngHabu) [inline config.yaml:68]: You need to run `make kustomize`

**Comment by bnsblue** [comment]
This one is only used by local execution of propeller. 

I don't think we need to change anything in Flyte if I am not mistaken. @wild-endeavor
